c++/src/kj/source-location.h

```diff
@@ -63,12 +63,17 @@ class SourceLocation {
   {}
 #endif
 
+  constexpr bool operator==(const SourceLocation& o) const {
```

Comment:
Hmm... I think this can only be exposed under `KJ_COMPILER_SUPPORTS_SOURCE_LOCATION` as otherwise all `SourceLocation` instances would be equal & that's a nonsensical result.

Reply:
Fixed in latest revision.